hw/ip/otbn/data/insns.yml

```diff
@@ -815,19 +819,26 @@ insns:
       - If `grs1_inc` is set, the value in `grs1` is incremented by the value WLEN/8 (one word).
       - If `grd_inc` is set, the value in `grd` is incremented by the value 1.
 
+      Only aligned (to WLEN sized regions) accesses are supported. Any unaligned access will result in an erorr.
+
       TODO: how to handle overflows?
     decode: |
       rd = UInt(grd)
       rs1 = UInt(grs1)
       offset = UInt(offset)
     operation: |
-      memaddr = GPR[rs1] + (offset / WLEN / 8)
+      memaddr_byte = GPR[rs1] + offset
       wdr_dest = GPR[rd]
 
-      WDR[wdr_dest] = LoadWlenWordFromMemory(memaddr)
-
       if grs1_inc and grd_inc:
-          raise Unsupported() \# prevented in encoding
+          raise Unsupported() # prevented in encoding
+      if memaddr_byte % (WLEN / 8) != 0:
```

Comment:
This is Python pseudo-code, so the "!= 0" isn't needed (and I think makes things harder to read).